### ISSUE TITLE
"At a glance" total user metric always displays "-"

### ISSUE THREAD

**Issue by theolundqvist** [parse-community/parse-dashboard#2502]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

The metric at the "apps" dashboard always shows a "-" on both total users and total installations:
![image](https://github.com/parse-community/parse-dashboard/assets/31588188/c8890755-f7a2-4fb5-a957-1ccde5f3a5ee)

I have a class called "_User" per the default with one user in right now on the testing DB. I have tried to search for anyone else having this issue but could not find any. Also tried going through the source code but did not find a solution. Have I missed a config setting or is it a bug?

### Steps to reproduce

This is my config. Database has the default "_User" class. On the readme the dashboard prints total users: 0
```js
const config = {
  apps: [
    {
      serverURL: env.publicServerURL,
      appId: env.appId,
      masterKey: env.masterKey,
      readOnlyMasterKey: env.readOnlyMasterKey,
      appName: env.appName,
      supportedPushLocales: ["en"],
      production: env.DB == "prod",
      iconName: "SuitClubs.svg",
    },
  ],
  users: [
    {
      user: "master",
      pass: env.dashWritePass,
    },
    {
      user: "moderator",
      readonly: true,
      pass: env.dashReadOnlyPass,
    },
  ],
  iconsFolder: "icons",
};

```

### Actual Outcome

`total users: -`

### Expected Outcome

`total users: 1`
### Environment

Dashboard
- Parse Dashboard version: `5.2.0`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Arc and Chrome`
- Browser version: `latest, but also last 1-2 years while i've used parse on different configurations`

Server
- Parse Server version: `6.2.2`
- Operating system: `macos Ventura 13.5.1`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local and fly.io and aws app runner and aws ec2`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `MongoDB Atlas prob. latest`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `MongoDB Atlas`

### Logs
Did not find any relevant logs.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.4.0-alpha.6](https://github.com/parse-community/parse-dashboard/releases/tag/5.4.0-alpha.6)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0-beta.1](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0)
